### PR TITLE
Pruning/Integrate new-developed pruning API with old one.

### PR DESCRIPTION
**Pull request by wenhuach21**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

Merge two pruning APIs to one.
API changed

### CODE REVIEW

**Comment by chensuyue** [comment]
Should we close https://github.com/intel/neural-compressor/pull/226 ?

**Comment by wenhuach21** [comment]
> Should we close #226 ?

please keep it for a while to have a reference

File: .azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt [hunk 1]
```diff
@@ -741,6 +743,7 @@ horovodrun
 hostfile
 Hounsfield
 howpublished
```
Comment (ftian1) [inline .azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt:745]: which file used this word? shall we use full word for it?

Comment (chensuyue) [reply .azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt:745]: This word used here: 
image

Comment (chensuyue) [reply .azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt:745]: I will update this readme in the last sprint.

**Pull request closed without merging**